src/common/components/Button.tsx

```diff
@@ -8,6 +8,7 @@ type Color = typeof colors[number];
 export interface CommonButtonProps {
   variant?: 'contained' | 'outlined' | 'text';
   color?: Color;
+  textColor?: Color;
```

Comment:
I have a suspicion that we may regret not sticking to `backgroundColor` and `color` here.

Reply:
I'm thinking this may be temporary until we can go in and set up text colors explicitly for the different button types (primary, secondary, etc.). We could consider keeping it around for special cases or overriding with css in special cases. I'm not sure which is better.